Source/WebCore/Modules/cookie-store/CookieStore.cpp

```diff
@@ -392,6 +402,23 @@ void CookieStore::set(CookieInit&& options, Ref<DeferredPromise>&& promise)
     Cookie cookie;
     cookie.name = WTFMove(options.name);
     cookie.value = WTFMove(options.value);
+
+    if (cookie.name.isEmpty()) {
```

Comment:
Isn't the spec not doing the `=` and special characters in the opposite order? This is web observable so ordering matters.

Reply:
You're right--the ordering is 
1. Invalid characters
2. empty name and '=' in value
3. empty name and empty value
I've fixed this in the latest iteration.